### ISSUE TITLE
Error matching terminal for CBVAV

### ISSUE THREAD

**Issue by jmarrec** [NREL/EnergyPlus#6544]
Issue overview
--------------

During regression testing of OpenStudio, I have a problem with the CBVAV system

```
   ** Severe  ** AirLoopHVAC:UnitaryHeatCool:VAVChangeoverBypass, "AIR LOOP HVAC UNITARY HEAT COOL VAVCHANGEOVER BYPASS 1", No matching air terminal found in the zone equipment list for zone = STORY 6 WEST PERIMETER THERMAL ZONE.
   **  Fatal  ** GetCBVAV: Errors found in getting AirLoopHVAC:UnitaryHeatCool:VAVChangeoverBypass input.
   ...Summary of Errors that led to program termination:
   ..... Reference severe error count=1
   ..... Last severe error=AirLoopHVAC:UnitaryHeatCool:VAVChangeoverBypass, "AIR LOOP HVAC UNITARY HEAT COOL VAVCHANGEOVER BYPASS 1", No matching air terminal found in the zone equipment list for zone = STORY 6 WEST PERIMETER THERMAL ZONE.
   ************* Warning:  Node connection errors not checked - most system input has not been read (see previous warning).
   ************* Fatal error -- final processing.  Program exited before simulations began.  See previous error messages.
```

I think the issue is here on [HVACUnitaryBypassVAV.cc#L1177](https://github.com/jmarrec/EnergyPlus/blob/develop/src/EnergyPlus/HVACUnitaryBypassVAV.cc#L1177)

This forwards to [DataZoneEquipment.cc#L1631](https://github.com/jmarrec/EnergyPlus/blob/develop/src/EnergyPlus/DataZoneEquipment.cc#L1631) which would expect the Zone Inlet Node Number, NOT the Zone Number.

I found several occurrences where `EquipList::getPrioritiesforInletNode` is called, and the zoneInlet assignment before the call makes a lot more sense (eg: [Furnaces.cc](https://github.com/jmarrec/EnergyPlus/blob/develop/src/EnergyPlus/Furnaces.cc#L3477) or [HVACMultiSpeedHeatPump.cc](https://github.com/jmarrec/EnergyPlus/blob/develop/src/EnergyPlus/HVACMultiSpeedHeatPump.cc#L658))

I lack the time or incentive to figure out how to handle the fact that `ZoneEquipConfig(zoneNum).InletNode` is an array in particular, but as a workaround if I replace line 1177 with 

`int zoneInlet = ZoneEquipConfig(zoneNum).InletNode(1); // TODO: temp workaround assumes only one inlet node
`
Then the simulation runs fine, which seems to corroborate that this is indeed the problem.

I'm attaching the IDF of the regression test that exhibits the Fatal error: [autosize_hvac.rb.idf.txt](https://github.com/NREL/EnergyPlus/files/1825726/autosize_hvac.rb.idf.txt)

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus: Version 8.9.0-9aca64c6a0

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
@jmarrec I see the problem. Pasteo from [this commit](https://github.com/NREL/EnergyPlus/commit/352495014578e5e26d957e550e5861fd1bfdcfb2#diff-81800c7b537d71afc0b63a3d290a7922R1199). It should have followed [this pattern](https://github.com/NREL/EnergyPlus/commit/352495014578e5e26d957e550e5861fd1bfdcfb2#diff-459b6d1c116d5661f0109a6583c024f8R1401). I'll post a fix today.

**Comment by jmarrec** [comment]
@mjwitte see the start of a fix + a small IDF snippet on https://github.com/NREL/EnergyPlus/pull/6545

feel free to push commits to this PR, or just take what you need and tell me to delete it and I'll do it asap (I'm gone for today).

thanks!

**Comment by Myoldmopar** [comment]
Closed via #6547
